File: Algoritmos e programação I/5. (14-04) Lista de Exercícios #2/Beecrowd_1071.py
'''
Beecrowd: Problema 1071
04/04/2024
Sophia Swiercoswski
'''

x = int(input())
y = int(input())
soma = 0

for impar in range(y + 1, x):
    if (impar % 2 != 0):
        soma += impar
print (soma)

File: Algoritmos e programação I/11. (21-08) Lista de Exercícios #4/Beecrowd_1071.cpp
/*
Beecrowd: Problema 1071
14/08/2024
Sophia Swiercoswski
*/

#include <iostream>
using namespace std;

int main() {
    // Ler dois valores inteiros
    int soma = 0, X, Y;
    cin >> X >> Y;

    // Percorrer sequência de números ímpares e somar
    for(int i = Y; i < X; i++) {
        if(i % 2 != 0) {
            i++;
            soma = i - 1;
        }
    }

    printf("%d\n", soma);
    return 0;
}